src/main/resources/application.yaml

```diff
@@ -72,3 +72,7 @@ mca:
   conditional-apis:
     case-assignments:
       enabled: ${MCA_CONDITIONAL_APIS_CASE_ASSIGNMENTS_ENABLED:true}
+
+notify:
+  api-key: ${NOTIFY_MCA_API_KEY:api_keys_api_keys_action_aca_test_api_key-18f9a06b-ee58-4973-937e-ccda99d55386-9fd5ef4a-e92d-4409-9ea7-8d5e49c0120f}
```

Comment:
This is not a prod key no doubt, but exposing an api key is not good even if it's not on prod.